02.create-and-explore/03.start-working-in-your-sandbox/14.creating-and-securing-your-domain/docs.md

```diff
@@ -0,0 +1,154 @@
+	- **Rez Temporary Certified**: A user can rez pre-approved entities or scripts from the Marketplace with a set lifetime that also defaults to 1 hour
+	- **Write Assets**: A user can add assets (models, audio, or other files) to your asset server (your domain's file storage space)
+	- **Ignore Max Capacity**: A user can enter a domain even when it has hit the specified capacity limit
+	- **Kick Users**: A user is allowed to remove other users from a domain
+	- **Replace Content**: A user can change the entire content set of a domain
+10. To save your domain settings, click on the “Download Domain Settings” button, which will save your settings to a JSON file
+	- You can upload these later using the “Upload Domain Settings” button  
+	![download-settings.png "Download Settings"]
+
+## Modifying Domain Content
+
+1. Click on the “Content” submenu in the server settings top bar to manage your content archives
+	- Associated actions:
+		- **Restore from here**: Resets the domain to that saved state
+		- **Download**: Compresses all of the content in your domain to a downloaded zip file
+		- **Delete**: Removes the archive from the saved archives list  
+	![content-archives.png "Content Archives"]
+2. Select your archive zip file and choose "Upload Content" to upload a content archive  
+	![upload-content.png "Upload Content"]
+3. Update paths for your domain
+	- When specified in Interface's “Go to” application in this format: “domain/pathname”, the user will be taken to a specific part of the domain instead of the default starting point  
+	![paths.png "Paths"]
+4. Set a script to always run on your domain by adding it to the "Persistent Scripts" table  
+	![persistent-scripts.png "Persistent Scripts"]
+5. Set an audio environment to create a "virtual stage"
+	- **Attenuation**: How much quieter sounds get over a distance
+		- The default domain attenuation is the amount of noise reduction that is enabled across the domain environment
+	- By setting a zone, you can specify axis aligned, 3D boundaries for audio environments that aren't applied to the whole domain
+		- **Attenuation coefficients**: How sound is transmitted between zones
+			- *When the attenuation coefficient is 0, there is no volume attenuation with someone in another zone; at 1, you won't be able to hear someone in another zone*
+	- **Reverb Settings**: Enables echo-like effects
+		- **Zone**: The zone that you defined earlier in the table
+		- **Reverb Decay Time**: How long you can hear an echo after the initial sound
+		- **Wet/Dry Mix**: The percentage mix of the reverb tail relative to the original "dry" signal  
+	![audio-environment.png "Audio Environment"]
+6. Set an entity filter on the whole domain:  
+	![filter-entities.png "Filter Entities"]  
+	To protect all entities in a domain while granting edit rights, copy the URL to a JavaScript filter script that follows this template:  
+	`prevent-all-deletes.js` by Brad Hefta-Gaub [^1]
```

Comment:
these are funky.  Let's make the whole thing be a link you can go to, right now it's just a little 1 you press next to it, same with the other one below

Reply:
these are supposed to be footnotes. is this style not supported in docs? https://rephrase.net/box/word/footnotes/syntax/